packages/deb_pkg.bzl

```diff
@@ -0,0 +1,31 @@
+def _impl(ctx):
+  args = [
+    "-source-file=" + ctx.file.source.path,
+    "-source-url=" + ctx.attr.source.base_url,
+    "-pkg-name=" + ctx.attr.pkg_name,
+    "-output-file=" + ctx.outputs.deb.path
+  ]
+
+  ctx.action(
+    executable = ctx.executable.package_getter,
+    inputs = [ctx.file.source],
+    arguments = args,
+    outputs = [ctx.outputs.deb],
+  )
+
+deb_pkg = rule(
```

Comment:
We want this to be a `repository_rule`, so that it can run in `WORKSPACE`, which brings with it new challenges (e.g. you can't depend on building the tool you plan to run).

Let's talk about this more once you are back, but what I do for `docker_pull` is publish a PAR file.
 - cloudbuild to produce it on release tags: https://github.com/google/containerregistry/blob/master/puller.yaml
 - where the binary is pulled down: https://github.com/bazelbuild/rules_docker/blob/master/docker/docker.bzl#L32
 - where the binary is used: https://github.com/bazelbuild/rules_docker/blob/master/docker/pull.bzl#L67

Once this is working, we should consider where the best place for this rule would be.  @damienmg FYI